Lombiq.Tests.UI.Samples/Tests/BasicVisualTests.cs

```diff
@@ -0,0 +1,173 @@
+using Lombiq.HelpfulLibraries.Common.Utilities;
+using Lombiq.Tests.UI.Attributes;
+using Lombiq.Tests.UI.Constants;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+using Shouldly;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Processing;
+using System.Globalization;
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Lombiq.Tests.UI.Samples.Tests;
+
+// In this basic test we will check visually the rendered content.
+public class BasicVisualTests : UITestBase
+{
+    public BasicVisualTests(ITestOutputHelper testOutputHelper)
+        : base(testOutputHelper)
+    {
+    }
+
+    // Checking that everything is OK with the branding on navbar of homepage.
+    // For this magic we are using ImageSharp.Compare package. You can find more info about it here:
+    // https://github.com/Codeuctivity/ImageSharp.Compare/blob/1.2.11/README.md.
+    [Theory, Chrome]
+    public Task AnonymousHomePageShouldExist(Browser browser) =>
+        ExecuteTestAfterSetupAsync(
+            context =>
+            {
+                context.SetViewportSize(CommonDisplayResolutions.HdPlus);
+
+                var testDumpFolderName = nameof(AnonymousHomePageShouldExist);
+                var testTempRootFolder = FileSystemHelper.EnsureDirectoryExists(Path.Combine("Temp", testDumpFolderName));
+
+                // We take a screenshot and save it to a temporary folder, and append temporary file to failure dump.
```

Comment:
And break this at 120 chars.

```suggestion
                // We take a screenshot and save it to a temporary folder, and append the temporary file to the failure dump.
```